lib/irc.js

```diff
@@ -633,6 +140,16 @@ Client.prototype.chanData = function(name, create) {
     return this.chans[key];
 };
 
+Client.prototype._parseCmd = function(message) {
+    var self = this;
+    var fn = parseCmd[message.command];
+    if (fn != undefined) {
+        if (typeof(fn) == 'function') fn(self, message);
```

Comment:
what's the advantage?
you would have to replace all the selfs in the original code
also, its adds overhead in performance
which is all ok, but what's the advantage?